assessments/projects/etcd/self-assessment.md

```diff
@@ -0,0 +1,434 @@
+## Table of contents
```

Comment:
Please provide a heading to this document in the format of "<project name> Security Self Assessment"